app/src/main/res/values/strings.xml

```diff
@@ -140,6 +140,10 @@
     <string name="pref_mouse_passthrough_summary_off">Use mouse events for local gestures</string>
     <string name="pref_mouse_passthrough_summary_on">Send mouse events directly to server</string>
     <string name="pref_hide_local_cursor">Hide local (Android) pointer</string>
+    <string name="pref_mouse_back">Map Mouse Back</string>
```

Comment:
This can simply be `Back button`.

Although, with current logic, "real" mouse back button is not mapped. For example, my mouse has left, right, wheel, forward, and back button. Its back button is not affected by current logic because back-click is reported via `onGenericMotionEvent`, and not as a `KeyEvent`.

As such, this preference may be confusing to users if they try to map the real back button. Are you planning to use this preference for middle click ?

Reply:
I have changed it.
Yes, I'm planning to use this preference for making three fingers tap on my Samsung keyboard cover touchpad act as middle click.